src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -603,11 +607,21 @@ ucs_status_t uct_dc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
         }
 
         ep->flags |= UCT_DC_MLX5_EP_FLAG_FLUSH_CANCEL;
+        sn         = txwq->sw_pi;
+        /* post NOP operation which will be handled to reset DCI */
```

Comment:
`/* post NOP operation which will complete with error, to trigger DCI reset. Otherwise, DCI could be returned to poll in error state */`

Reply:
updated comment